dbt_copilot_helper/commands/dns.py

```diff
@@ -423,17 +421,14 @@ def create_required_zones_and_certs(domain_client, project_client, subdomain, ba
     return create_cert(project_client, domain_client, subdomain, cert_zone_id)
 
 
-def _get_zone_id_or_abort(client, zone):
+def _get_zone_id_or_abort(hosted_zones, zone):
     """Zones have the '.' suffix."""
-    response = client.list_hosted_zones_by_name()
-    hosted_zones = {hz["Name"]: hz for hz in response["HostedZones"]}
     if zone not in hosted_zones:
         click.secho(
-            f"The base domain: {zone} does not exist in your AWS domain account {hosted_zones}",
+            f"The hosted zone: {zone} does not exist in your AWS domain account {hosted_zones}",
             fg="red",
         )
         exit()
-    click.secho(f"Found hosted zone {zone}", fg="green")
```

Comment:
This was just noise in the output and didn't really add anything, so removed it.